src/content/docs/en/editor-setup.mdx

```diff
@@ -53,6 +53,12 @@ In addition to local editors, Astro also runs well on in-browser hosted editors,
 
 See [the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/user-guide/) for more information on how to install and set up ESLint for your project.
 
+### Stylelint
+
+[Stylelint](https://stylelint.io/) is a popular linter for CSS. [A community maintained configuration](https://github.com/ota-meshi/stylelint-config-html) provides Astro support. 
+
+Installation instructions, editor integration, and additional information can be found in the project's [README.](https://github.com/ota-meshi/stylelint-config-html#cd-installation)
```

Comment:
```suggestion
[Stylelint](https://stylelint.io/) is a popular linter for CSS. [A community maintained Stylelint configuration](https://github.com/ota-meshi/stylelint-config-html) provides Astro support. 

Installation instructions, editor integration, and additional information can be found in the project's README.
```
This looks great, thank you! I just updated the linked text to be more explicit (so that it had added context) and avoided a second link that was almost the same as the first link.  :) Too much linkage can look distracting, and if someone clicks the first link, they'll find the info from the second link easily enough.